### DISCUSSION TITLE
Console spits status constantly

### DISCUSSION THREAD

**Discussion by guydvir2** [arendst/Tasmota#11293] [Q&A] [answered]
Hi,
I flashed a generic `tasmota.bin` file onto a Sonoff Dual R2, and set it as a shutter ( following the instructions ).
Device works as expected, but its console spits status messages 0-13 all the time ( fractions of a second apart ).

What is the reason for that ?

**Comment by arendst** [comment, marked as answer]
TasmoAdmin

**Reply by guydvir2** [reply]
You are right, TasmoAdmin is installed. Is is OK ? is there a way to stop it ? it fills up console I can't follow what happens

**Reply by guydvir2** [reply]
OK, closing TasmoAdmin's web page stops it :)

**Reply by sfromis** [reply]
[TDM](https://github.com/jziolkowski/tdm) recommended instead.
